config/VehicleConfigurations.xml

```diff
@@ -80,6 +80,10 @@ You can define the following custom settings:
     soon as the the front reaches the row start/field edge, avoiding unworked patches with irregular implement
     work areas such as a plow.
 
+- modName: Name of the .zip file (without '.zip')
+    In case a Mod have the same .xml file name as a Vanilla vehicle or implement, add the name of the .zip file to
+    prevent a false entry. Note: Only 2 of the same .xml files can be added.
+

```

Comment:
In case a Mod has the same .xml filename for the vehicle/implement as a default giants vehicle/implement, add the name of the .zip file to prevent a conflicts. For example: "FS22_exampleMod"
 Note: Only 2 of the same .xml files can be added.